operators.js

```diff
@@ -88,24 +86,34 @@ const merge = (streams) => {
  * @return {TransformStream}
  */
 const switchMap = (fn, options = { pairwise: true }) => {
+    let lastRead = undefined;
     return new TransformStream({
         // mousedown
-        transform(chunk, controller) {
+        async transform(chunk, controller) {
```

Comment:
on all transform calls the inner stream is recreated, like the doc: https://www.learnrxjs.io/learn-rxjs/operators/transformation/switchmap